packages/vuetify/types/services/icons.d.ts

```diff
@@ -9,8 +9,9 @@ export interface VuetifyIconOptions {
    * mdi: <a href="https://materialdesignicons.com">MDI</a>
    * fa: <a href="https://fontawesome.com/get-started/web-fonts-with-css">FontAwesome 5</a>
    * fa4: <a href="">FontAwesome 4</a> TODO: link
+   * faSvg: <a href="https://fontawesome.com/how-to-use/on-the-web/using-with/vuejs">FontAwesome SVG</a>
    */
-  iconfont: 'mdi' | 'mdiSvg' | 'md' | 'fa' | 'fa4' // TODO: camelCase
+  iconfont: 'mdi' | 'mdiSvg' | 'md' | 'fa' | 'fa4' | 'faSvg' // TODO: camelCase
```

Comment:
team: this TODO doesn't seem to be valid any longer

Reply:
I think it was for `iconfont` -> `iconFont`

Reply:
Appreciate your work on this!  I created a PR against your branch to address this change. https://github.com/basselAhmed/vuetify/pull/1

Reply:
Thanks, @albernhagen merged it.